### PR TITLE
Revert "Make Unique Index Name Unique #6516"

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
Reverts #6516

This reverts commit 22f9797390be4e883bd37ec3e03b8b55f05dbd96, reversing changes made to 6561bd709f8b7960ec858afa1b2a0723e683b73f.

Conflicts:
	user_guide_src/source/changelogs/v4.3.0.rst

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

**Review by MGatner — APPROVED** [review]
Please let @sclubricants have a final word (here or other thread) before merging.

**Review by michalsn — APPROVED** [review]
Thanks, kenjis.

**Comment by sclubricants** [comment]
Looks good. Lets get this merged so I can get the setting manual names PR in.

**Pull request merged**
